src/Builder.cpp

```diff
@@ -52,12 +52,18 @@ struct SortEntry {
 // reallocations
 constexpr size_t minSortEntriesAllocation = 32;
 
+
+#ifndef NO_THREADLOCALS
```

Comment:
I think it is better to use macro names prefixed with the library name here, again to make them unambiguous.
```suggestion
#ifndef VELOCYPACK_NO_THREADLOCALS
```
Additionally just wondering what the problem is with requiring thread-locals. Is there any issue with them?

Reply:
The issue is that they do not play well with memory contexts, they will point to freed memory once the memory context gets destroyed.